docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -515,6 +518,25 @@ Equivalent to the Podman `--health-cmd` option.
 Set an interval for the healthchecks. An interval of disable results in no automatic timer setup.
 Equivalent to the Podman `--health-interval` option.
 
+### `HealthLogDestination=`
+
+Set the destination of the HealthCheck log. Directory path or events_logger (none use container state file)
+Equivalent to the Podman `--health-log-destination` option.
```

Comment:
Maybe add a sentence at the end here saying "the default is `none`" so it's very clear - I initially missed the (default) bit below.

Reply:
Also, `none` doesn't feel like the right name. Maybe `local`?